
This knocks five minutes of the build time on my 2.4Ghz x86,
probably hours on really slow machines. There is no semantic
difference.

Help ghc find libgmp.a

--- ../ghc-4.08.2/distrib/hc-build.orig	Wed Aug  9 21:09:43 2000
+++ ../ghc-4.08.2/distrib/hc-build	Tue Aug  5 10:50:09 2003
@@ -16,6 +16,8 @@
 GhcLibHcOpts = -O
 SRC_HAPPY_OPTS += -c
 GhcLibWays=
+SplitObjs=NO
+SRC_HC_OPTS += -L${LOCALBASE}/lib
 END
 
 # touch happy generated files; so that in non-bootstrapping mode for
